setup.py

```diff
@@ -25,9 +25,13 @@
     python_requires=">=3.7",
     install_requires=[
         "python-dateutil>=2.7.0",
+        "ciso8601~=2.2.0",
         'typing_extensions >= 3.7; python_version < "3.8"',
     ],
-    extras_require={"validation": ["jsonschema>=3.0"], "orjson": ["orjson>=3.5"]},
+    extras_require={
+        "validation": ["jsonschema>=3.0", "pytest"],
```

Comment:
```suggestion
        "validation": ["jsonschema>=3.0"],
```

Don't need `pytest` as a distributed dependency.